common/ip-packet-requests/src/lib.rs

```diff
@@ -15,6 +16,18 @@ pub struct IPPair {
     pub ipv6: Ipv6Addr,
 }
 
+impl IPPair {
```

Comment:
imho to match other convention of other related types, probably rename capitalization to `IpPair`?